README.md

```diff
@@ -120,6 +120,14 @@ listen_addr = '127.0.0.1:8232'
 parallel_cpu_threads = 0
 ```
 
+It is recommended to use [adityapk00/lightwalletd](https://github.com/adityapk00/lightwalletd) because that is used in testing.
+
+If using [zcash/lightwalletd](https://github.com/zcash/lightwalletd.git):
+- note that it will require a zcash.conf file 
+- `rpcuser` and `rpcpassword` are required but ignored by Zebra
+- when using a non-default port, use `rpcport=28232` and `rpcbind=127.0.0.1`
+- when using testnet, use testnet=1
```

Comment:
```suggestion
- note that it will require a zcash.conf file:
  - `rpcuser` and `rpcpassword` are required but ignored by Zebra
  - when using a non-default port, use `rpcport=28232` and `rpcbind=127.0.0.1`
  - when using testnet, use `testnet=1`
```